scripts/generate-integration-template-tests.bash

```diff
@@ -8,6 +8,13 @@ pushd () {
 popd () {
     command popd "$@" > /dev/null
 }
+if [[ "$(uname)" == "Darwin" ]]; then
```

Comment:
This change was already pushed into `main` so maybe you need to pull the latest to not see this changed reflected in this PR